lib/debugger.coffee

```diff
@@ -23,19 +23,28 @@ class ProcessManager extends EventEmitter
     result[key(e)] = value(e) for e in env.split(";")
     return result
 
-  start: (file) ->
+  startActiveFile: () ->
+    @start true
+
+  start: (withActiveFile) ->
+    startActive = withActiveFile
     @cleanup()
       .then =>
+        packageJSON = require @atom.project.resolvePath('package.json')
```

Comment:
Instead of using `resolvePath`, is that better to do something like find the first package.json from current `.js` file? Assuming is will be a atom project looks like(I saw a lot of people do this):

```
-- app1/
| -- package.json
| -- lib/
  | -- something.js
-- app2
| -- package.json
| -- /lib
  | -- app.js
```